modules/misc/test/tests/ad_arrhenius_material_property/tests

```diff
@@ -0,0 +1,23 @@
+[Tests]
+  [exact]
+    type = 'CSVDiff'
+    input = 'exact.i'
+    csvdiff = 'exact_out.csv'
+    issues = '#22384'
+    design = 'ArrheniusMaterialProperty.md'
+    requirement = 'The system shall compute a generic material property based on an Arrhenius model that matches an analytic solution of that equation while using the automatic differentiation system'
+  []
+  [exact-jac]
+    type = 'PetscJacobianTester'
+    input = 'exact.i'
+    run_sim = 'True'
+    cli_args = "Outputs/active=''"
+    prereq = exact
+    petsc_version = '>=3.9'
+    ratio_tol = 3e-7
+    difference_tol = 1
+    issues = '#847'
```

Comment:
Is this a BISON issue number? If so, it should be changed or removed here.

Reply:
Not a BISON issue number. It's a reference to the issue I made in MOOSE requesting the functionality